packages/financial-templates-lib/test/clients/FinancialContractEventClient.js

```diff
@@ -173,15 +174,15 @@ contract("FinancialContractEventClient.js", function(accounts) {
                     timelockLiveness: 86400, // 1 day
                     rewardRatePerSecond: { rawValue: "0" },
                     proposerBondPercentage: { rawValue: "0" },
-                    maxFundingRate: { rawValue: convertSynthetic("0.00001") },
```

Comment:
IDK why this was not a `toWei()` before but we make teh assumption that funding rates are always in wei units, so should not be converting to the synthetic precision